diana/byoc/test.py

```diff
@@ -136,3 +142,47 @@ def test_mlperf_tiny_mobilenet(run, tmp_path):
 
 def test_mlperf_tiny_dae(run, tmp_path):
     run_full_network(run, tmp_path, mlperf_tiny.relay_dae.create_model)
+
+
+###
+#   Full network tests, network defined in ONNX
+###
+
+
+def run_onnx_x86(onnx_file: str, directory: str):
+    """Compile and run an ONNX model on x86
+    """
+    # ensure .npy files are present to get numeric checking too
+    onnx_dir = os.path.dirname(onnx_file)
+    assert os.path.exists(os.path.join(onnx_dir, 'input.npy'))
+    assert os.path.exists(os.path.join(onnx_dir, 'output.npy'))
+
+    mod, params = driver.parse_onnx_input(onnx_file)
+    d = driver.X86Driver(mod, params, directory)
+    d.tvm_compile()
+    d.gcc_compile(gcc_opt=0)
+    d.run(gdb=False)   # raises exception on failure
+
+
+def test_mlperf_tiny_ds_cnn_onnx_x86(run, tmp_path):
+    run_onnx_x86('test_data/export_dscnn/DSCNN_QL_NOANNOTATION.onnx', tmp_path)
+
+
+def test_mlperf_tiny_resnet_onnx_x86(run, tmp_path):
```

Comment:
Can you either:
* Add a conditional skip to the following tests if git lfs is not used? And mention in the repo that LFS is required for this type of tests
* Add git lfs to the requirements in the README.md? And require them for all tests

Reply:
I'll update the README.md that they have to `git lfs pull` before running `pytest -v test.py`.